### PR TITLE
Use HTTPS for assets

### PR DESCRIPTION
**Pull request by mathiasbynens**
This makes https://sbstjn.github.io/timesheet.js/ work without any mixed content errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,11 +9,12 @@
...
-    <link href="http://fonts.googleapis.com/css?family=Oxygen+Mono:400,600,800" rel="stylesheet" type="text/css" />
-    <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" type="text/css" />
-    <link href="http://fonts.googleapis.com/css?family=Signika+Negative:300,400,600,700" rel="stylesheet" type="text/css" />
-    <link href="stylesheets/all-4d3cb721.css" rel="stylesheet" type="text/css" />
+    <script src="javascripts/timesheet-ec6cdbbf.js"></script>
+    <script src="javascripts/main-d80740b0.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Oxygen+Mono:400,600,800" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Signika+Negative:300,400,600,700" rel="stylesheet" />
+    <link href="stylesheets/all-4d3cb721.css" rel="stylesheet" />
```
Comment (mathiasbynens) [inline index.html]: Note that I also removed `type=text/javascript` and `type=text/css` as [they are redundant](https://mathiasbynens.be/notes/html5-levels#level-1).

**Comment by sbstjn** [comment]
Thanks!

**Comment by mathiasbynens** [comment]
Could you please change the link at the top of this page to the HTTPS version?

Below the repo description:

**Comment by sbstjn** [comment]
Oh thanks, I changed it …

**Comment by mathiasbynens** [comment]
:+1:

**Pull request merged**